SQL/Archive/2018-02-15_AddCustomizableCRFsUnderLinksInImagBrowser.sql

```diff
@@ -0,0 +1,6 @@
+-- Add Imaging Browser to Imaging Modules
+INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'linkedInstruments', 'Instruments that the users want to see linked from Imaging Browser', 1, 1, 'instrument', ID, 'Imaging Browser Links to Instruments', 7 FROM ConfigSettings WHERE Name="imaging_modules";
```

Comment:
Can you change the name to something link "ImagingBrowserLinkedInstruments"? "linkedInstruments" as a generic config name used throughout LORIS isn't really clear.